source/docs/software/pathplanning/system-identification/viewing-diagnostics.rst

```diff
@@ -1,53 +1,63 @@
 Viewing Diagnostics
 ===================
 
-The first diagnostic to look at is the r-squared - it should be somewhere north of ~.9. If it is significantly lower than this, there is likely a problem with your identification data.
+Goodness-of-Fit Metrics
+-----------------------
+
+There are three numerical accuracy metrics that are computed with this tool: acceleration r-squared, simulated velocity r-squared, and standard deviation of the simulated velocity error.
+
+.. image:: images/viewing-metrics.png
+   :alt: Analysis accuracy metrics
+
+The `r-squared <https://en.wikipedia.org/wiki/Coefficient_of_determination>`__ metric refers to the goodness of fit of the tool's `OLS (Ordinary Least Squares) fit <https://en.wikipedia.org/wiki/Ordinary_least_squares>`__.  The `RMSE (Root Mean Squared Error) <https://en.wikipedia.org/wiki/Root-mean-square_deviation>`__ is the error (in the measured units) between the collected velocities and the predicted velocities from a simulation based on the feedforward gains.
 
-The next diagnostic to look at is the RMSE to make sure that the error is reasonable and within your desired constraints.
+The acceleration r-squared is the fraction of the variance in measured acceleration (used as the independent variable in the SysId regression) explained by the linear model.  This can be quite variable variable, because acceleration is very susceptible to system noise.  Assuming the other fit metrics are acceptable, values near ``1`` indicate an "ideal" mechanism with few disturbances, while values near ``0`` indicate a noisy mechanism with substantial physical vibrations/losses.
```

Comment:
If this is not so important, maybe it shouldn't be listed first.

Reply:
I'm not sure I'd say it's not important - it's definitely a useful measurement.  The ordering in the docs matches the ordering on the UI.